src/marqo/tensor_search/api.py

```diff
@@ -54,6 +56,7 @@ def generate_config() -> config.Config:
 
 
 _config = generate_config()
+_embed = embed_module.Embed(config=_config)
```

Comment:
The standard pattern is to add `Embed` to config, rather than pass config to it. Similar to how `IndexManagement` etc are in config

Reply:
done